src/js/renderers/ArrayControlRenderer.js

```diff
@@ -94,7 +100,7 @@ export const GutenbergArrayRenderer = (props) => {
             ...prevScreenContent,
             [`${route}`]: {
                 component: ( {
-                    ...props
+                    ...ownControlProps
```

Comment:
So the props passed to Navigator context is control props without connect to jsonforms context